app/routes/index.tsx

```diff
@@ -74,15 +74,15 @@ export const loader: LoaderFunction = async ({ request }) => {
 
 function CalendarPage() {
   const { setEvents, events: eventList, view } = useStore((state) => state);
-  const { events, error } = useLoaderData();
-
+  const { events, error, calendarId } = useLoaderData();
   useEffect(() => {
     if (error === null && events.length > 0) {
       // TODO: show a  different message if events are not present in the given date range
-
+      sessionStorage.setItem('calendarId', calendarId)
```

Comment:
any reason to use calendarId instead of the global storage?

Reply:
Have used zustand to store the calendar ID now